debian/control

```diff
@@ -349,6 +349,8 @@ Description: Backup Archiving Recovery Open Sourced - common tools
 
 
 
+
```

Comment:
I guess this is an accident. The debian/control file should not be a part of this PR, please remove.